src/App.js

```diff
@@ -33,10 +66,19 @@ class Image extends React.Component {
         window.removeEventListener("scroll", this.handleScroll);
     }
 
+    deleteCat = () => {
+        const countToDelete = this.state.countToDelete
+        this.setState((prevState) => {
+            const buffArray = prevState.images.slice(countToDelete, prevState.images.length);
```

Comment:
Minor: you can just name this array `images`. This name is not used in the scope of this function and it would be easier to read.